src/main/java/com/aws/greengrass/builtin/services/configstore/ConfigStoreIPCEventStreamAgent.java

```diff
@@ -270,7 +274,14 @@ public UpdateConfigurationResponse handleRequest(UpdateConfigurationRequest requ
 
                 Object value = request.getValueToMerge();
 
-                Topics serviceTopics = kernel.findServiceTopic(serviceName);
+                logger.atInfo().kv(SERVICE_NAME, serviceName).log(String.format(
```

Comment:
should not info log. This is at best debug, and possibly even trace.

Reply:
Do not use String.format, our logger will correctly format everything for you. Use the logger's formatting.

Reply:
The intent is we want any component-to-component configuration updates to be logged, so that if an unintended configuration update happens it can be traced back to the requesting component. If that is done at debug or trace, those logs would not be saved under normal circumstances.

However, I think there may be a better place for info level logging that meets that need. Junfu mentioned that info level logging may already be present for IPC operations. And I was also considering having the logging happen somewhere in the Authorization flow. I plan on looking more into those, unless you have other thoughts on this

Reply:
We should not be logging everything at info. Info is the default level that we have enabled. IPC is used far too often to have this type logged at info level, it must be debug.

Reply:
I see; sounds good. I suppose for the use case of tracing unwanted actions, it's reasonable to enable debug level logging first, and then monitor for it.

I'll make this log here debug. And if I find a better place for it when I'll update it at that point.